src/koop/tokenMarkLogicAuthentication.js

```diff
@@ -36,12 +36,11 @@ function authorize(req) {
                 err.code = 403;
                 reject(err);
             }
-            log.debug("setting req.marklogicUsername to " + decoded.sub);
             let username = decoded.sub;
-            req.marklogicUsername = username;
+            req.markLogicClientCacheKey = username;
```

Comment:
Just did required renaming in here. I'd like to clean this up and ensure any error handling blocks are being tested, but in a future PR.